test/integration/deployment/util.go

```diff
@@ -351,3 +349,26 @@ func (d *deploymentTester) waitForDeploymentUpdatedReplicasLTE(minUpdatedReplica
 func (d *deploymentTester) waitForDeploymentWithCondition(reason string, condType v1beta1.DeploymentConditionType) error {
 	return testutil.WaitForDeploymentWithCondition(d.c, d.deployment.Namespace, d.deployment.Name, reason, condType, d.t.Logf, pollInterval, pollTimeout)
 }
+
+func (d *deploymentTester) listPods(selector labels.Selector) ([]v1.Pod, error) {
```

Comment:
Since we only list pods that are owned by new RS (but not old RSes), name it to `listUpdatedPods` would be more clear.